ls1intum/Artemis
# Make yarn test work on all platforms

### Motivation and Context
`yarn test` and similar commands do not work on Windows because the shell script is called:
![image](https://user-images.githubusercontent.com/11130248/133827988-da43c7c3-967d-4462-8672-30ef39343483.png)

### Description
- Switch to calling `jest.js` directly. The shell script does the same, but seems to handle CYGWIN in a special way.
- The script now works on Ubuntu WSL and Windows, and I cannot see any reason why it should not work. We require jest to be present as module anyway.
- No information I found online indicates the other approach is necessary

### Steps for Testing
- You don't know of any reason why we shouldn't switch to `jest.js`?
- Run the new `yarn test` commands on your system and check that they work.

### Review Progress
- [ ] Tested that it works on Linux
- [ ] Tested that it works on Windows
- [ ] Tested that it works on MacOS
